cmd/wallet-web/src/pages/chapi/Signup.vue

```diff
@@ -36,31 +36,25 @@
           </ul>
         </div>
         <!--Trustbloc Sign-up provider div -->
-        <div class="col-span-1 md:block sm:object-none sm:object-center md:py-18 lg:px-4 xl:py-14 xl:px-4 lg:py-14 ">
+        <div class="col-span-1 md:block sm:object-none sm:object-center md:py-18 lg:px-4 xl:py-14 xl:px-4 lg:py-14">
           <div class="flex justify-center items-center lg:hidden md:hidden xl:hidden xs:block sm:block ">
             <img class="h-4 w-8 mr-1 mt-4 md-4 my-2" src="@/assets/img/logo.svg">
             <h2 class="font-bold text-8xl lg:text-2xl md:text-4xl sm:text-6xl xs:text-8xl">TrustBloc</h2>
           </div>
           <div class="text-center items-center mb-10">
-            <h1 class="font-bold text-3xl 2xl:text-lg lg:text-2xl md:text-4xl sm:text-6xl xs:text-8xl text-neutrals-softWhite">Sign up. It's free!
+            <h1 class="font-bold text-3xl 2xl:text-4xl lg:text-2xl md:text-4xl sm:text-6xl xs:text-8xl text-neutrals-softWhite">Sign up. It's free!
             </h1>
           </div>
-           <md-card-content v-if="loading" style="margin: 10% 20% 10% 30%">
-              <beat-loader :color="'white'" :size="20"></beat-loader>
-              <transition name="fade" mode="out-in">
-                <div class="text-white" style="padding-top: 10px;" :key="messageNum">{{messages[messageNum]}}</div>
-              </transition>
-            </md-card-content>
-            <div class="flex 2xl:mx-8 xl:mx-8 lg:mx-4 md:mx-4 mx-4 -mt-6">
-              <div class="w-full px-8 mb-8 2xl:mr-8 xl:mr-8 lg:mr-8 md:mr-4 sm:mx-2 sm:mb-8 md:mb-8 items-center">
+            <div class="flex 2xl:m-8 xl:mx-8 lg:mx-4 md:mx-4 mx-4 align-middle content-center">
+              <div class="w-full px-8 mb-8 2xl:m-8 xl:m-8 lg:m-8 md:m-4 sm:mx-2 sm:mb-8 md:mb-8">
                 <button class="2xl:flex 2xl:flex-wrap lg:flex lg:flex-wrap md:flex md:flex-wrap text-xs 2xl:text-xs xl:text-xs lg:text-sm
```

Comment:
Please, use the button class defined in theme to avoid duplicate styles here as well. We can add flexbox there if it's universal for all buttons too 👍🏼